app/components/InputSearch.js

```diff
@@ -51,13 +53,13 @@ class InputSearch extends React.Component<Props> {
   };
 
   render() {
-    const { theme, placeholder = "Search…" } = this.props;
+    const { theme, t } = this.props;
 
     return (
       <InputMaxWidth
         ref={(ref) => (this.input = ref)}
         type="search"
-        placeholder={placeholder}
+        placeholder={t("Search...")}
```

Comment:
Need to retain the ability to pass `placeholder` as a prop here. It's an ellipses (…) at the end of the string too, not three periods.

Reply:
ups, my bad ... fixed